context/app/static/js/components/home/ExploreTools/ExploreTools.tsx

```diff
@@ -1,13 +1,113 @@
-import React from 'react';
+import React, { useState } from 'react';
 
-import { Typography } from '@mui/material';
+import Grid from '@mui/material/Unstable_Grid2';
+import { useTheme } from '@mui/material/styles';
+import { entityIconMap } from 'js/shared-styles/icons/entityIconMap';
+import { ToolsCard } from './ToolsCard';
+import { CardGridContextProvider } from './CardGridContext';
+import { ToolDescription } from './ToolDescription';
+
+const workspaceCTAAuthenticated = {
+  ctaText: 'View Your Workspaces',
+  ctaLink: '/workspaces',
+  ctaIcon: <entityIconMap.Workspace />,
+};
+
+const workspaceCTAGuest = {
+  ctaText: 'Sign in',
+  ctaLink: '/login',
+};
+
+const cards = [
+  {
+    title: 'Analyze data in Workspaces',
+    src: `${CDN_URL}/v2/explore-tools/workspaces.png`,
```

Comment:
I plan to add webp versions of these images with the implementation of the updated Hero.